src/msg/msg_types.h

```diff
@@ -19,6 +19,8 @@
 
 #include <netinet/in.h>
 
+#include <fmt/format.h>
```

Comment:
how about adding these specifier to a dedicated header file, like `msg_fmt.h`? so we don't have to include `fmt/format.h` header file everywhere?

Reply:
my concern is the build-time dependency. i am worried about the build time. as your change could have a profound impact of the amount of code to be build, as you are not only adding `fmt/format.h` to some very popular header files and also add this template specialization to it.